pkg/server/image_list.go

```diff
@@ -17,14 +17,31 @@ limitations under the License.
 package server
 
 import (
-	"errors"
-
 	"golang.org/x/net/context"
 
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
 // ListImages lists existing images.
+// TODO add filters
```

Comment:
Follow the `TODO(name): XXX` format which we are using in k8s?
And the same with following. It would be great if you could add a priority with the TODOs, so that people could know which to work on first.